packages/web-components/fast-components/src/disclosure/fixtures/disclosure.html

```diff
@@ -0,0 +1,98 @@
+<fast-design-system-provider use-defaults>
+    <h1>Disclosure</h1>
+    <fast-card
+        class="class-override"
+        style="--card-width: 400px; --card-height: auto; --elevation: 24; padding: 12px;"
+    >
```

Comment:
Is there any value in having this render in a card, or is it carryover from the card file?

Reply:
Just to make a separation between all components in the preview.

Reply:
Gotcha - this may end up causing issues as cards are soon going to extend design system provider which would cause the visual appearance to push based on the card background. We might want to simply leverage the heading OR add a slight bit of margin to each element using a style block:

```html
<style>
    fast-disclosure {
        margin-top: 12px;
    }
</style>
```